test/rest_api/test_storage_service.py

```diff
@@ -531,7 +539,7 @@ def make_snapshot_tag(name):
                         assert snapshots[after_alter_keyspace_tag]['total'] == snapshots[after_flush_tag]['total'], f"snapshots after alter-keyspace should have the same data as after flush: {snapshots}"
                         assert snapshots[after_cleanup_tag]['total'] == 0, f"snapshots after clean should have no data: {snapshots}"
 
-@pytest.mark.xfail(run=False, reason="rest_api suite doesn't support tablets yet (#17338), run test manually")
+@pytest.mark.xfail(run=False, reason="fails bacause of #17452, should be enabled once it's fixed")
```

Comment:
@lkshminarayanan if #17465 is merged after this one, please enable this test

Reply:
@Deexie Issue #17452 is now fixed (via PR #17502).